### DISCUSSION TITLE
Advanced by default?

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#83] [Q&A]
Is there a way we can have the advanced options shown by default? Thanks.

**Comment by abartesaghi** [comment]
A possible solution will be for the state of the advanced option to persist across parameter windows. We will look into this. Is there a specific parameter that you use often and would prefer not to be advanced?

**Reply by asarnow** [reply]
There's quite a few I would like to see & remember, especially when reading the block parameters again and again. The one I think might be best to actually move is the gres field on the main Resource tab.
